### DISCUSSION TITLE
Add instructions on how to get Nextcloud AIO setup with Rancher

### DISCUSSION THREAD

**Discussion by Monsterray** [nextcloud/all-in-one#1961] [Ideas] [answered]
I have been trying to get Nextcloud AIO working with my Rancher installation and haven't been able to get it running at all. I've used all of the docker compose settings but am getting an error about permissions for the docker.sock file. The error I'm getting is reported below. Could you please make a guide on how to get this up and running?

> Trying to fix docker.sock permissions internally...
> Creating docker group internally with id 999
> It seems like you did not give the mastercontainer the correct name?
> Using a different name is not supported!

**Comment by szaimen** [comment, marked as answer]
Hi, since Rancher is built around kubernetes IIRC, I don't think that it is possible to get AIO running there since AIO is deeply coupled to the Docker API and depends on e.g. containers having specific names.

However you could check if you can get its containers running with https://github.com/nextcloud/all-in-one/tree/main/manual-install or https://github.com/nextcloud/all-in-one/tree/main/nextcloud-aio-helm-chart.

If that should be successful, please let me know and I will add it to the official documentation. Thanks!

**Reply by Monsterray** [reply]
I added the nextcloud/all-in-one git repository to the repositories on rancher and tried to install the nextcloud-aio app and it gives me the error below. Any help on what this means would be greatly appreciated!

> Secret "helm-operation-8r9g6" is invalid: [data[values-Nextcloud AIO Helm Chart-4.3.4.yaml]: Invalid value: "values-Nextcloud AIO Helm Chart-4.3.4.yaml": a valid config key must consist of alphanumeric characters, '-', '_' or '.' (e.g. 'key.name', or 'KEY_NAME', or 'key-name', regex used for validation is '[-._a-zA-Z0-9]+'), data[Nextcloud AIO Helm Chart-4.3.4.tgz]: Invalid value: "Nextcloud AIO Helm Chart-4.3.4.tgz": a valid config key must consist of alphanumeric characters, '-', '_' or '.' (e.g. 'key.name', or 'KEY_NAME', or 'key-name', regex used for validation is '[-._a-zA-Z0-9]+')]

**Reply by szaimen** [reply]
Hi, not sure which of these values gives the wrong regex: https://github.com/nextcloud/all-in-one/tree/main/nextcloud-aio-helm-chart/values.yaml

**Reply by hadifarnoud** [reply]
the helm chart url is not found 404
correct url: https://github.com/nextcloud/all-in-one/tree/main/nextcloud-aio-helm-chart
